hyundai_kia_connect_api/VehicleManager.py

```diff
@@ -80,7 +85,13 @@ def __init__(
         self.vehicles: dict = {}
 
     def initialize(self) -> None:
-        self.token: Token = self.api.login(self.username, self.password)
+        login_sig = insp.signature(self.api.login)
+        kwargs = {}
```

Comment:
Could this be done without doing kwargs and just use optional arguments added instead?